content/inside-rust/program-management-update-2025-08.md

```diff
@@ -0,0 +1,270 @@
+
+Once this next round settles down, he will open the RFC (it will likely be several documents each focusing on a different stage of the effort).
+
+## Rust for Linux
+
+[Rust for Linux](https://rust-for-linux.com/) is an ongoing initiative to be able to write Linux kernel code in Rust. The motivation is memory safety, a modern language and bring in more contributors due to the combination of the two.
+
+The focus in this second half of 2025 is to bring unstable features the project is using into stable Rust. Linux takes stability and backwards compatibility very seriously and will not rely on features that can change from release to release or even be removed entirely.
+
+### RFC: pass pointers to `const`
+
+Rust has support for [inline assembly](https://doc.rust-lang.org/nightly/rust-by-example/unsafe/asm.html), via the `asm!` macro for example. Assembly is as close as one can get to writing machine code directly. This lets you do things the language doesn't have direct support such as accessing CPU registers or reading/writing values to device-specific segments in memory that allow control their behavior (this is often used when writing drivers).
+
+The macro lets you interpolate constant values into the assembly code you've written, similar to how you can interpolate values to a format string or macro:
+
+```rust
+println!("The meaning of life, universe and everything: {}", 42);
+```
+
+But constant values only get you so far. A common thing when writing assembly is to be able to pass *pointer* values (e.g. pointer to a specific field of a struct you want to manipulate) around. But currently, [only integer constant expressions are allowed](https://doc.rust-lang.org/1.89.0/reference/inline-assembly.html#r-asm.operand-type.supported-operands.sym).
+
+[Alice Ryhl](https://github.com/Darksonn) opened an RFC that [allows specifying pointers in the `const` operand](https://github.com/rust-lang/rfcs/pull/3848) too. Conceptually (and in assembly specifically) pointers could be thought of as just numbers that are interpreted as addresses to memory (although in Rust, the story is far more complex and [pointers are not the same thing as integers](https://doc.rust-lang.org/std/ptr/index.html#provenance)).
+
+This is now ready for feedback from the Lang team so I've opened a [design issue](https://github.com/rust-lang/lang-team/issues/347)  and got it scheduled for review and discussion.
+
+### Field projections
+
+[field-projections]: https://rust-lang.github.io/rust-project-goals/2025h2/field-projections.html
+
+We also had a design meeting on [Field Projections][field-projections]. When you have a type behind e.g. `Box` or `Rc`, you can access its field "directly" as if the wrapper/pointer type wasn't there:
```

Comment:
I'd say it's better to have an example using references `&`/`&mut`, since that's the more fundamental type. They also already have field projections built into the language, where `Box` and `Rc` do not.